.github/workflows/linux-ci.yml

```diff
@@ -12,6 +12,7 @@ jobs:
         sudo apt install cmake libcurl4-openssl-dev libopenal-dev libssl-dev \
           libogg-dev libopus-dev libspeex-dev uuid-dev libvpx-dev \
           libvorbis-dev qtbase5-dev
+        sudo pip install -r ${GITHUB_WORKSPACE}/requirements.txt
```

Comment:
Too bad the resource.dat is never built on linux-ci since it's part of the plClient build :trollface:

Reply:
Not currently, but hopefully that will change.  I know the scripts run on there because I've done so. :trollface:

Reply:
Since the requirements file allows specifications based on environment (such as OS), it makes sense to add this to both build systems for consistency (and to watch for accidental breakages).